examples/_client_i3s.rb.example

```diff
@@ -15,4 +15,6 @@ require 'pry'
 puts "Connected to Image Streamer at #{@client.url}\n\n"
 
 # Instance variables for examples:
-@plan_script_name = 'Plan_Script_1'
```

Comment:
You renamed the `@plan_script_name` to `'Plan_Script_Name'` here, but `examples/image-streamer/api300/build_plan.rb` is still using it as if it was named `'Plan_Script_1'`